robot_modeling/DQ_SerialManipulator.m

```diff
@@ -215,48 +161,7 @@ function set_effector(obj,effector)
             p(6) = h(2)*h(8)+h(6)*h(4)+h(1)*h(7)+h(5)*h(3);
             p(7) = h(3)*h(8)+h(7)*h(4)-h(1)*h(6)-h(5)*h(2);
             p(8) = h(4)*h(8)-h(3)*h(7)-h(2)*h(6)+h(1)*h(5);
-        end
-        
-        function J = raw_pose_jacobian(obj,q,ith)
```

Comment:
This method does not depend on any convention. Therefore, it should be maintained as an abstract method to enforce that all subclasses implement it.

Reply:
Hi @bvadorno, 

I think this is the same case as a previous (now resolved) conversation. The method raw_pose_jacobian() is an abstract method in `DQ_SerialManipulator` and is implemented in `DQ_SerialManipulatorDH`.

Best regards, 

Juancho